app/src/components/Card/CardPreview/index.tsx

```diff
@@ -40,56 +39,97 @@ function CardPreview({ card, onClick }: ICardPreviewProps) {
+                      <span className="text-[#46434e] text-sm font-semibold font-['Rubik'] leading-[18.90px]">
+                        @MattD
+                      </span>
+                    </div>
                   </div>
                 ))}
-          </div>
-          {tasks?.length > 4 ? (
-            <span className='ml-3 flex items-center h-full underline text-sm'>
-              +{tasks.length - 4}
-            </span>
-          ) : null}
+              </div>
+              {tasks.length > 4 && (
+                <span className='ml-3 flex items-center h-full underline text-sm'>
+                  +{tasks.length - 4}
+                </span>
+              )}
+            </div>
+          ) : (
+            <div className='flex-grow overflow-hidden'>
+              {card.description.map((line, index) => (
+                <div key={'description-' + line + index} className="text-[#46434e] text-sm font-normal font-['Rubik'] leading-tight overflow-hidden" style={{ display: '-webkit-box', WebkitLineClamp: 2, WebkitBoxOrient: 'vertical', overflow: 'hidden', textOverflow: 'ellipsis' }}>
+                  {line}
+                </div>
+              ))}
+            </div>
+          )}
         </div>
-        {/* <div className='grid grid-cols-5 mt-5'> */}
-        <div className='flex mt-5 sm:max-w-[390px] md:max-w-[300px] lg:max-w-[450px] '>
-          {!isEmpty(imageUrls) ? (
-            <div className='flex col-span-2 '>
-              {imageUrls?.map((url, idx) => (
-                <Image key={'image-' + url + '-' + idx} className='w-12' src={url} alt='' />
+        <div className="mt-auto w-full">
+          <div className="h-[56px] w-full pr-2 bg-white rounded-sm justify-start items-center gap-1 inline-flex">
+            <div className="justify-start items-center flex w-full">
+              {(imageUrls ?? []).slice(0, 4).map((url, index) => (
+                <img key={index} className="w-14 h-14 relative rounded-sm border border-[#efefef]" src={url} />
               ))}
-              {card?.files?.length > 5 ? (
-                <span className='ml-3 flex items-center h-full font underline text-sm ml-12'>
-                  +{card.files.length - 5}
-                </span>
-              ) : null}
+              {(imageUrls ?? []).length > 4 && (
+                <div className="w-14 h-14 bg-white rounded-sm border border-[#efefef] justify-center items-center flex">
+                  <span className="text-[#46434e] text-sm font-normal font-['Monaspace Argon Var']">
+                    +{(imageUrls ?? []).length - 4}
+                  </span>
+                </div>
+              )}
             </div>
-          ) : null}
+          </div>
+          <div className="mt-2 flex justify-start items-center gap-2">
+            <AvatarGroup>
+              {(card.users as IUser[])?.map((user, index) => (
+                <Avatar key={index} src={user.avatarUrl} alt={user.name} radius="xl" size="sm" />
```

Comment:
The key should be an unique value (for example user id)

Reply:
Actually we have [component ](https://github.com/prspktvs/dun/blob/d1791944525a763d1894a0e69e0d6319eb89e150/app/src/components/ui/Avatar.tsx)for avatars